base_local_planner/src/local_planner_util.cpp

```diff
@@ -75,7 +75,7 @@ void LocalPlannerUtil::reconfigureCB(LocalPlannerLimits &config, bool restore_de
     nominal_limits_ = LocalPlannerLimits(config);
     ROS_INFO("Nominal limits tolerance: %f", nominal_limits_.xy_goal_tolerance);
   }
-  updateLimits();
```

Comment:
what is the impact of removing this line?

Reply:
The updateLimits function takes in the current local plan to modify the current speed limits. When the node is initialized, the reconfigureCB is called with and empty path causing errors. This shouldn't impact the robot because updateLimits is called whenever the controller is called at 15hz.